CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Introduction
+
+The purpose of this document is to show contributors to the Modernisation Platform code the best practises and testing that should be undertaken. Code should be in Terraform and tests, if coded, in Go.
+
+## Code of Conduct
+
+The Ministry of Justice [Code of Conduct](https://github.com/ministryofjustice/.github/blob/main/CODE_OF_CONDUCT.md) is used to govern the project and those participating.
+Please report unacceptable behavior to the maintainers or administrators.
+
+## Standards
+
+We follow the [MoJ Technical Guidance](https://technical-guidance.service.justice.gov.uk/#moj-technical-guidance),
+which complements the Service Standard,
+and also the MoJ [Cyber and Technical Security Guidance](https://security-guidance.service.justice.gov.uk/#cyber-and-technical-security-guidance).
+
+# Modernisation Platform Documentation
+To access the modernisation documentation follow this [MP Documentation](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform).
+For the general terraform you can access here [Terraform](https://user-guide.modernisation-platform.service.justice.gov.uk/runbooks/terraform.html)
+If you are interested in running Terraform locally try [Running Terraform Plan Locally](https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/running-terraform-plan-locally.html#running-terraform-plan-locally)
+
+These can be located by running a search in the documentation [MP Documentation](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform). 
+
+# Coding examples
```

Comment:
A few spelling mistakes here, but personally I would point customers at our module template, and a module that you think is particularly worthwhile. The code in `ministryofjustice/modernisation-platform-environments` is relevant to modernisation-platform users and is specific to that repository.

Reply:
This is kind of mentioned in the coding section below this but I have put a comment in place